jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -136,14 +137,26 @@ public Builder setEntrypoint(@Nullable List<String> entrypoint) {
       return this;
     }
 
+    /**
+     * Sets the user name (or UID) and optionally the user group (or GID) which the process in the
```

Comment:
Wording suggestion: `Sets the user and group to run the container as. {@code user} can be a username or UID along with an optional groupname or GID. The following are all valid: {@code user}, {@code uid}, {@code user:group}, {@code uid:gid}, {@code uid:group}, {@code user:gid}.`

Reply:
done